clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/job/KubectlJobExecutor.java

```diff
@@ -589,6 +586,26 @@ private Void patch(KubernetesV2Credentials credentials, KubernetesKind kind, Str
     return null;
   }
 
+  private StreamConsumer parseManifestList(List<KubernetesManifest> manifestList) {
+    return (InputStream is) -> {
+      JsonReader reader = new JsonReader(new InputStreamReader(is, StandardCharsets.UTF_8));
```

Comment:
if we're always immediately assuming UTF_8 encoding and wrapping the inputstream in `StreamConsumer`s, would it make more sense to have them be `ReadableConsumer=` or `ReaderConsumer` and only have to deal in characters and not bytes?

If they do need to stay as InputStream consumers, does the encoding need to come from `Charset.defaultCharset()` to avoid any issues on systems that are not using UTF-8 character encoding for standard out. Admittedly, I think its unlikely that in practice this is being run on any systems that that would be the case on, so feel free to ignore

Reply:
I think it makes sense to remove the responsibility for converting characters to bytes from the implementation and just give them a `Reader`.  If at some point we need a consumer that handles a stream of bytes, we can refactor to account for that, but I don't forsee that so it doesn't make sense to make this overly general.

I'd imagine non-UTF-8 is pretty unlikely, though it's probably better to just use the default encoding, which I've done as part of moving this translation to `JobExecutorLocal`.